testing/mocks.ts

```diff
@@ -30,7 +40,9 @@ export function newMockExecutionContext(overrides?: Partial<MockExecutionContext
     timezone: 'America/Los_Angeles',
     invocationToken: v4(),
     fetcher: {
-      fetch: sinon.stub(),
+      fetch: sinon.stub<[FetchRequest], Promise<FetchResponse>>().callsFake(async r => {
+        throw new Error(`Unhandled fetch: ${r.method} ${r.url}`);
```

Comment:
Updating `newMockExecutionContext` to throw by default (rather than just return an empty value).